hack/verify.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Jetstack Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Make the zonal cluster resource definiton from the regional cluster resource
+# definition. This helps to keep the two definitions the same, except for the
+# presence of the region or zone property.
+
+set -e
+
+./hack/verify-zonal.sh
+./hack/verify-terraform.sh
```

Comment:
Probably best to use a REPO_ROOT variable to reference these paths, so that these scripts can be run from outside of the root of the repository?

`REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..`

https://github.com/jetstack/cert-manager/blob/ff19e63ea451eace615cc41cc49019c470835b16/hack/verify-vendor.sh#L21